css/header.css

```diff
@@ -53,4 +53,37 @@
     .header__promo-text {
         font-family: "Intro Script R H2 Base";
     }
+}
+@media (min-width:1080px){
+    .header__burger{
+        display: none;
+    }
+    .header__nav{
+        display: block;
+        width: 62%;
+        
+    }
+    .header__nav-wrapper{
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+    .header__link{
+        font-size: 13px;
+        text-transform: uppercase;
+        font-family: "Circe Regular";
+        color: #2d2829;
+    }
+    .header__link:hover{
+        color: #f61f5e;
+    }
+    .header__phone{
+        margin-left: 10%;
```

Comment:
Please avoid using percantage for margin, padding

Reply:
done